packages/client/public/imtr/transformed/37mYBbnidDABYfjiM.json

```diff
@@ -35,7 +35,7 @@
           }
         },
         "dummy": {
-          "requiredDecisions": ["#_060c0745-57aa-4c20-8604-2a0104355bcb"],
+          "requiredDecisions": ["6fc55eb5e1a97153f46ee6d2c909e61e"],
```

Comment:
Making sure the references from `requiredInputs` and `requiredDecisions` was the 'tricky' part.